WORKSPACE

```diff
@@ -152,6 +152,22 @@ dev_env_tool(
     win_tool = "maven-3.6.1",
 )
 
+dev_env_tool(
+    name = "sbt_dev_env",
+    nix_include = ["bin/sbt"],
+    nix_label = "@sbt_nix",
+    nix_path = "bin/sbt",
+    tool = "sbt",
+    win_include = [
+        "bin",
+        "boot",
+        "conf",
+        "lib",
+    ],
+    win_path = "bin/sbt",
+    win_tool = "sbt-1.2.8",
```

Comment:
For this to work on Windows you'd have to add `sbt` to `.dadew` and `dev-env/windows/manifests`.

Reply:
Thanks! :-)